search/utils.py

```diff
@@ -0,0 +1,366 @@
+"""
+Search functions.
+"""
+
+from __future__ import unicode_literals
+
+from collections import defaultdict
+from lxml import etree
+import logging
+
+from django.conf import settings
+from elasticsearch.helpers import bulk
+from elasticsearch.exceptions import NotFoundError
+from elasticsearch_dsl.connections import connections
+from elasticsearch_dsl import Search, Mapping, query
+
+from statsd.defaults.django import statsd
+
+from learningresources.models import get_preview_url, LearningResource
+from search.search_indexes import get_course_metadata
+
+log = logging.getLogger(__name__)
+
+DOC_TYPE = "learningresource"
+INDEX_NAME = settings.HAYSTACK_CONNECTIONS["default"]["INDEX_NAME"]
+URL = settings.HAYSTACK_CONNECTIONS["default"]["URL"]
+CONN = connections.create_connection(hosts=[URL])
+PAGE_LENGTH = 10
```

Comment:
Would it make sense to move this to `lore.settings`, or is this something that we probably wouldn't need to tweak?